### PR TITLE
Fix bf passthrough flashing for DIY RX

### PR DESCRIPTION
**Pull request by greenbigfrog**

### CODE REVIEW

**Comment by greenbigfrog** [comment]
This should work. Gonna solder a RX tomorrow and test.

What was the reasoning for having `runpython.py` in the first place? I can't really think of where it's being used.

File: src/platformio.ini [hunk 1]
```diff
@@ -336,8 +336,8 @@ src_filter = ${env:DIY_900_RX_ESP8285_SX127x_via_UART.src_filter}
 upload_speed = 74880
 upload_protocol = custom
 upload_command =
-	python python/runpython.py $PYTHONEXE python/BFinitPassthrough.py 74880
-	python $PROJECT_PACKAGES_DIR/framework-arduinoespressif32/tools/esptool.py --no-stub -b 74880 -c esp8266 --before no_reset --after soft_reset write_flash 0x0000 $SOURCE
+	python $PROJECT_DIR/python/BFinitPassthrough.py 74880
```
Comment (cruwaller) [inline src/platformio.ini:339]: replace `74880` by `$UPLOAD_SPEED` to avoid unnecessary copy of the same value. 
will be easy to try different values :)

Comment (greenbigfrog) [reply src/platformio.ini:339]: Bloddy hell... I went back and fixed it in the esptool part to use `$UPLOAD_SPEED` but totally missed it for init passthrough apparently?

Comment (greenbigfrog) [reply src/platformio.ini:339]: wait. it's completly gone... gonna fix it

**Comment by AlessandroAU** [comment]
Looks good, can someone running windows do a quick verification test? I don't have my hardware with me atm

**Review by cruwaller — APPROVED** [review]

**Comment by JyeSmith** [comment]
Doesnt work for the DIY_2400_RX_ESP8285_SX1280_via_BetaflightPassthrough.

**Comment by AlessandroAU** [comment]
@JyeSmith did it ever work for you?

**Comment by cruwaller** [comment]
This needs to be picked into `develop` as well 👌

**Pull request merged**